Rules/CommonScripts/CinematicCommon.as

```diff
@@ -0,0 +1,303 @@
+enum ImportanceRank
+{
+	capturing_vehicle,
+	lit_keg,
+	player_near_tent,
+	builder_near_flag,
+	capturing_hall,
+	missing_flag
+}
+
+enum ComparisonType
+{
+	cinematic_importance,
+	x_position,
+	y_position
+}
+
+float calculateZoomLevelH(uint height)
+{
+	return 360.0f / Maths::Max(height, 1.0f);
+}
+
+float calculateZoomLevelW(uint width)
+{
+	Driver@ driver = getDriver();
+	float ratio = float(driver.getScreenHeight()) / float(driver.getScreenWidth());
+	return calculateZoomLevelH(width * ratio);
+}
+
+void calculateZoomTarget(float distX, float distH)
+{
+	float zoomW = calculateZoomLevelW(distX * 1.8f);
```

Comment:
calculateZoomLevelW calls calculateZoomLevelH. Can we just combine this into a single function that returns a Vec2 instead of double calling calculateZoomLevelH?